cvat/apps/engine/serializers.py

```diff
@@ -74,11 +73,12 @@ class ServerFileSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.ServerFile
         fields = ('file', )
-
-    def to_internal_value(self, data):
+    @staticmethod
+    def to_internal_value(data):
         return {'file': data}
 
-    def to_representation(self, instance):
+    @staticmethod
+    def to_representation(instance):
```

Comment:
By specification the method should have self. Please find another way to fix the particular issue.